### PR TITLE
Rg.Plugins.Popup Compatibility with iOS 13 Beta #459

### PR DESCRIPTION
**Pull request by DevEddy**
Fix by @michaelstonis

#466 #459

### CODE REVIEW

**Comment by luizvnegrini** [comment]
@DevEddy Do you have any idea how often approvals are? PRs have been open for more than months and have not been made or merged yet and this correction made you seem very urgent.

**Comment by DevEddy** [comment]
@luizvnegrini Sorry, I have no idea

**Comment by SolidSoft-Lda** [comment]
While there a nuget package today or should I download the source code and compile ?

**Comment by DevEddy** [comment]
@SolidSoft-Lda You can use this source to test the changes made in this pr: https://ci.appveyor.com/nuget/rg-plugins-popup-p6l5elugev99

It should be the version 1.1.5.214

**Comment by SolidSoft-Lda** [comment]
@DevEddy 
Thank you very much !

**Comment by slang25** [comment]
We've been testing this fix in our app and it's working great 🎉 Thanks @DevEddy

File: src/Rg.Plugins.Popup.IOS/Impl/PopupPlatformIos.cs [hunk 1]
```diff
@@ -36,15 +40,19 @@ public async Task AddAsync(PopupPage page)
...
             var renderer = page.GetOrCreateRenderer();
 
-            var window = new PopupWindow
+            var window = new PopupWindow();
+
+            if (IsiOS13OrNewer)
             {
-                BackgroundColor = Color.Transparent.ToUIColor()
-            };
+                _windows.Add(window);
```
Comment (rotorgames) [inline src/Rg.Plugins.Popup.IOS/Impl/PopupPlatformIos.cs]: Why do you keep a window in the list? What reason?

Comment (DevEddy) [reply src/Rg.Plugins.Popup.IOS/Impl/PopupPlatformIos.cs]: @michaelstonis

Comment (rotorgames) [reply src/Rg.Plugins.Popup.IOS/Impl/PopupPlatformIos.cs]: @DevEddy @MichaelRumpler Ok I got it. You try to keep windows in memory while pages are visible. But ios has to do it automatically. It looks like a bug of Xamarin.iOS or iOS.Do you still get this issue even with stable releases of Xamarin.iOS 13 and VS4Mac 8.3?

Comment (rotorgames) [reply src/Rg.Plugins.Popup.IOS/Impl/PopupPlatformIos.cs]: So I just have replicated it. It looks like GC problem. GC removes UIWindow even if it's a kay window. Anyway, this solution looks pretty well. 

Thanks, guys.

**Comment by rotorgames** [comment]
Good work guys. 

@michaelstonis @DevEddy

**Pull request merged**